components/Account/NewsletterSubscriptions.js

```diff
@@ -70,11 +70,8 @@ class NewsletterSubscriptions extends Component {
               {subscriptions.map(({ name, subscribed, isEligible }) => (
                 <p key={name}>
                   <Checkbox
-                    checked={subscribed}
-                    disabled={
-                      (!isEligible && !subscribed) ||
-                      mutating[name]
-                    }
+                    checked={subscribed && isEligible}
+                    disabled={!isEligible || mutating[name]}
```

Comment:
Why? Once the membership resumes they would automatically be re-subscripted. Current logic is that they can unsub while not eligible but can't resub. After this change people could come under the impression that we lost their preference.

Reply:
I found UI better to describe current state of your subscription, hereby moving its meaning from "subscribed, but not in segment = no mail" to "no subscription = no mail" and by losing eligibility losing subscription, and not accounting for loss (and restoring) preferences here.

So much for why part.

Reverting however to previous logic.

Reply:
I agree that we can do better here. We can tackle this separately later.